sass/components/_btn.scss

```diff
@@ -2,3 +2,10 @@
   line-height: 1;
   padding: .25rem 1rem;
 }
+
+.btn-primary:disabled,
+.btn-primary.is-disabled {
```

Comment:
i think it was easiest to add right to this selector -- let me know what u think!